joint_limits:

  shoulder_pan_joint:
    has_position_limits: true
    min_position: -1.658
    max_position: 1.4835
    #    has_velocity_limits: true
    #max_velocity: 2.0
    #has_acceleration_limits: true
    #max_acceleration: 5.0
    #has_jerk_limits: true
    #max_jerk: 100.0
    #has_effort_limits: true
    #max_effort: 1.0

  shoulder_pitch_joint: 
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    #has_velocity_limits: true
    #max_velocity: 2.0
    #has_acceleration_limits: true
    #max_acceleration: 5.0
    #has_jerk_limits: true
    #max_jerk: 100.0
    #has_effort_limits: true
    #max_effort: 1.0

  elbow_pitch_joint:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0

  elbow_roll_joint:
    has_position_limits: true
    min_position: -14835.
    max_position: 1.658

  wrist_pitch_joint:
    has_position_limits: true
    min_position: -1.396
    max_position: 1.7453
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0

  gripper_roll_joint:
    has_position_limits: true
    min_position: 0
    max_position: 3.14159
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0

  finger_joint1:
    has_position_limits: true
    min_position: 0
    max_position: 0.025
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0

  finger_joint2:
    has_position_limits: true
    min_position: -0.025
    max_position: 0
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0
